.github/workflows/label-check.yaml

```diff
@@ -0,0 +1,19 @@
+name: Labels
```

Comment:
```suggestion
name: Check PR labels
```

Reply:
I did it this way, because it makes the check read as:

```
Labels / ensure type label
```

With the suggestion it would be:

```
Check PR labels / ensure type label
```